README.md

```diff
@@ -53,7 +89,7 @@ map.on('load', () => {
 })
 
 // Create whatever HTML element you want as Cluster
-const clusterMode = (element: HTMLDivElement, props: MapGeoJSONFeature['properties'], size?: number): void => {}
+const clusterRender = (element: HTMLDivElement, props: MapGeoJSONFeature['properties'], size?: number): void => {}
```

Comment:
clusterRenderFn ?

Reply:
it's the function itself here, not the option. I think we don't need to specify the `fn` statement in it's name as we know already it's a function. It make sense on the option name as it indicates we are expecting a Function to be passed to this option.
What do you think ?

Reply:
Right